Dockerfile-alpine.template

```
@@ -132,8 +132,6 @@ RUN set -eux; \
 		/usr/local/share/doc \
 		/usr/local/share/man \
 	; \
-	find /usr/local -name '*.a' -delete; \
-	\
 	postgres --version
```

This empty line helps when we're doing diffs between Alpine / Debian, so I'd prefer to keep it, but otherwise this is great. :+1:

```suggestion
	; \
	\
	postgres --version
```

---

Do I need to modify my PR to not delete empty lines?